parsec/core/backend_connection/cmds.py

```diff
@@ -48,12 +48,7 @@
     device_cancel_invitation_serializer,
     device_create_serializer,
 )
-from parsec.core.types import (
-    UnverifiedRemoteUser,
-    UnverifiedRemoteDevice,
-    UnverifiedRealmRole,
-    EntryID,
-)
```

Comment:
What's an "unverified remote"?

Reply:
Old concept that stores both the certificate (the binary data) and a timestamp from when the data was received.
The idea was to validate the certificate, which end up in creating a `VerifiedRemoteDevice` structure.

Now all this stuff is replaced by the `DeviceCertificateContent` class